pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1272,17 +1273,18 @@ func (n *NetworkPolicyController) syncAppliedToGroup(key string) error {
 				appGroupNodeNames.Insert(pod.Spec.NodeName)
 			}
 			for _, extEntity := range externalEntities {
-				if extEntity.Spec.ExternalNode == "" {
+				entityKey := externalnode.GenerateEntityNodeKey(extEntity)
```

Comment:
this should be nodeKey or entityNodeKey, but not entityKey?

Reply:
updated.